frontend/src/pages/auth/components/auth-layout/styles.module.scss

```diff
@@ -46,7 +45,8 @@
 }
 
 .scrollableFormWrapper {
-  flex-basis: 553px;
+  flex-basis: 33%;
+  min-width: 500px;
```

Comment:
Not sure if min-width: 500 would be suitable for narrow screens

Reply:
[85b212f](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/372/commits/85b212f5e0c376445d779dfbaf157519586135e3)